target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -353,6 +353,10 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "2:lan:1" "3:lan:3" "4:lan:2"
 		;;
+	ubnt,acb-ac)
+		ucidef_add_switch "switch0" \
+			"0@eth0" "2:lan:1" "3:lan:2" "4:wan" "5:lan:3" "6@eth1"
```

Comment:
It's common to "group" lan and wan entries to make them easier to see.

Reply:
thank you adjusted.
would something like this be fine?

`"0@eth0" "6@eth1" "2:lan:1" "3:lan:2" "5:lan:3" "4:wan"`

i don't think that i can utilize `ucidef_set_interface_wan` here.